CONTRIBUTING.md

```diff
@@ -0,0 +1,22 @@
+
+# Contributing to the paper (or papers) about the R course
+
+Adding content to the repository:
+
+1. Have your own fork or create a branch from the master `paper-rcourse` repo.
+2. Work on a defined issue that was assigned to you or that you self-assigned.
+3. Submit a PR specific to the issue to close it. If you are still working on it,
+include `WIP:` in the title of your PR.
+
+Adding issues and dealing with labels, milestones, and assigning:
+
+- Anyone can create an issue. Anyone can self-assign themselves to an issue.
+- Only @lwjohnst86 and @mbonsma will manage labels, milestones, and projects, as 
+well as assigning others to tasks.
+
+Reviewing PR:
+
+- Anyone can review or comment on a PR.
+- All authors are responsible for commenting on and approving the PR with the final
+version of the paper, before we submit to the journal.
+- We need at least one reviewer to approve the PR before it can be merged.
```

Comment:
`at least one reviewer to approve any PR before it can be merged`

It sounds like this last line is referring to the final PR. Maybe also switch the order of this line and line 20.